docs/download.md

```diff
@@ -4,8 +4,40 @@ title: Download
 nav_order: 3
 ---
 
+[RAPIDS Accelerator For Apache Spark provides](https://github.com/NVIDIA/spark-rapids) a set of plugins for Apache Spark that leverage GPUs to accelerate processing via the RAPIDS libraries and UCX.
+
+[RAPIDS cuDF](https://github.com/rapidsai/cudf) project is the cornerstone for [RAPIDS Accelerator For Apache Spark provides](https://github.com/NVIDIA/spark-rapids).
+
+CUDA and NVIDIA Driver need to be installed on the machine where CUDA-capable GPU is attached.
+
+Spark job needs 2 jars -- "RAPIDS Accelerator For Apache Spark" jar, and "RAPIDS cuDF" jar based on corresponding CUDA&NVIDIA Driver version.
```

Comment:
I'd like to see this section be more prose-like, e.g.:

The RAPIDS Accelerator For Apache Spark provides a set of plugins for Apache Spark that leverage GPUs to accelerate Dataframe and SQL processing. The accelerator is built upon the RAPIDS cuDF project and UCX.  (Ideally `RAPIDS cuDF project` and `UCX` would be links to give these projects shoutouts.)

The RAPIDS Accelerator requires each worker node in the cluster to have CUDA installed.  (Maybe link to https://developer.nvidia.com/cuda-toolkit here).  
The RAPIDS accelerator consists of two jars, a plugin jar along with the RAPIDS cudf jar, that is either preinstalled in the Spark classpath on all nodes or submitted with each job that uses the RAPIDS Accelerator.  (Maybe link to the getting started documentation here.)

Reply:
Made the suggested change in the latest commit.